packages/extension-sdk-react/README.md

```diff
@@ -96,21 +90,22 @@ The Looker SDK can be accessed as follows:
 
 The extension context exposes the following properties:
 
-- `coreSDK` - SDK version 3.1 (kept for backwards compatability)
-- `core31SDK` - SDK version 3.1
-- `core40SDK` - SDK version 4.0
+- `coreSDK` - SDK version 4.0
 
 The following global access methods are available:
 
-- `getCoreSDK()` - SDK version 3.1 (kept for backwards compatability)
-- `getCore31SDK()` - SDK version 3.1
 - `getCore40SDK()` - SDK version 4.0
 
 There is no restriction on which SDK can be used within an extension, none, one or all of the above can be used interchangeably, context or global access. The one caveat is that it is recommended that the Looker version support SDK 4.0 if the 4.0 SDK is used (the results may be unpredictable otherwise).
 
+If you require access to SDK version 3.1, use `<ExtensionProvider/>` or `<ExtensionProvider2 type={Looker31SDK}/>`. SDK version 3.1 will then globally be available through the following methods:
```

Comment:
is this meant to read `<ExtensionProvider40 type={Looker40SDK} />`?

Reply:
No. The point of ExtensionProvider40 is so that a type is not required on the provider and a generic  is not required when getting the context.

This 
```
<ExtensionProvider2 type={Looker40SDK}>
. . .
  const extensionContext =
    useContext<ExtensionContextData2<Looker40SDK>>(ExtensionContext2)
```
Becomes
```
<ExtensionProvider40>
. . .
  const extensionContext =
    useContext(ExtensionContext40)
```

Reply:
understood